test/Templates.Test/Helpers/AspNetProcess.cs

```diff
@@ -74,15 +74,26 @@ public AspNetProcess(ITestOutputHelper output, string workingDirectory, string p
             {
                 var dllPath = publish ? $"{projectName}.dll" : $"bin/Debug/{framework}/{projectName}.dll";
                 _process = ProcessEx.Run(output, workingDirectory, DotNetMuxer.MuxerPathOrDefault(), $"exec {dllPath}", envVars: envVars);
+                var listeningUrlString = GetListeningUrlString(output);
+                _listeningUri = new Uri(listeningUrlString, UriKind.Absolute);
```

Comment:
Since in both cases where `GetListeningUrlString` is called, we immediately wrap the return value in a `new Uri`, would it be cleaner to have `GetListeningUrlString` itself return the `Uri` instance? (Presumably also rename it to something like `GetListeningUri`).